app/javascript/hyacinth_v1/components/shared/forms/FormButtons.js

```diff
@@ -21,7 +21,7 @@ function FormButtons(props) {
       </Col>
 
       <Col sm="auto">
-        <ResponsiveSubmitButton saveData={onSave} formType={formType} />
+        <ResponsiveSubmitButton key={`${formType}-submit`} saveData={onSave} formType={formType} />
```

Comment:
Why is the key attribute necessary here if this element doesn't have any siblings?

Reply:
It's not - it's debugging cruft.

Reply:
```suggestion
        <ResponsiveSubmitButton saveData={onSave} formType={formType} />
```